security/pfSense-pkg-acme/Makefile

```diff
@@ -0,0 +1,97 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-acme
+PORTVERSION=	0.1
+CATEGORIES=	security
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	PiBa-NL
+COMMENT=	pfSense package acme
+
+USE_PHP= ftp
```

Comment:
USE_PHP alone is deprecated. keep this line replacing space by TAB to separate = and value and add the following line before it:
```
USES=	php
```